doc/man1/tsget.pod

```diff
@@ -7,26 +7,26 @@ tsget - Time Stamping HTTP/HTTPS client
 =head1 SYNOPSIS
 
 B<tsget>
-B<-h> server_url
-[B<-e> extension]
-[B<-o> output]
+B<-h> I<server_url>
+[B<-e> I<extension>]
+[B<-o> I<output>]
 [B<-v>]
 [B<-d>]
-[B<-k> private_key.pem]
-[B<-p> key_password]
-[B<-c> client_cert.pem]
-[B<-C> CA_certs.pem]
-[B<-P> CA_path]
-[B<-r> file:file...]
-[B<-g> EGD_socket]
-[B<request...>
+[B<-k> I<private_key.pem>]
+[B<-p> I<key_password>]
+[B<-c> I<client_cert.pem>]
+[B<-C> I<CA_certs.pem>]
+[B<-P> I<CA_path>]
+[B<-r> I<files>]
+[B<-g> I<EGD_socket>]
+[I<request> ...]
 
 =head1 DESCRIPTION
 
 The B<tsget> command can be used for sending a time stamp request, as
 specified in B<RFC 3161>, to a time stamp server over HTTP or HTTPS and storing
 the time stamp response in a file. This tool cannot be used for creating the
-requests and verifying responses, you can use the OpenSSL B<ts(1)> command to
+requests and verifying responses, you can use the L<openssl-ts(1)> command to
 do that. B<tsget> can send several requests to the server without closing
```

Comment:
Repalce ```B<tsget>``` with "this command"

Reply:
I suppose I should go through them all... "cause you're right, there's a *lot* of repeats like that...